highfive-toolfile.spec

```diff
@@ -0,0 +1,21 @@
+### RPM external highfive-toolfile 1.0
+Requires: highfive
+%prep
+
+%build
+
+%install
+
+mkdir -p %{i}/etc/scram.d
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/highfive.xml
+<tool name="highfive" version="@TOOL_VERSION@">
+  <info url="https://github.com/BlueBrain/HighFive"/>
+  <client>
+    <environment name="HIGHFIVE_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INCLUDE" default="$HIGHFIVE_BASE/include"/>
+  </client>
+  <runtime name="ROOT_INCLUDE_PATH" value="$INCLUDE" type="path"/>
+</tool>
```

Comment:
@mrodozov , please build locally and checkif highfix headers use `boost, hdf5 eigen or xtensor` then please make sure to add their dependency here too e..g 
```
<use name="boost"/> 
<use name="eigen"/>
```

Reply:
It only had hdf5 and boost headers. I wonder why xtensor and eigen are in requirements now